### PR TITLE
Review of maintenance_releases.pt

### PR DESCRIPTION
**Pull request by IldySilva**
referencing  #770

### CODE REVIEW

**Comment by IldySilva** [comment]
@pedropark99 , is this Pr Well done? i still facing some dificulties to understand the contribuitions flow

**Comment by pedropark99** [comment]
Yes! This PR is well done because it is the First Review of the translated content. In other words, since you took the lead, now, the second review should try to add changes to this PR in specific.

I (or any other person who wants to take the second review) can add changes to this PR in specific in two ways:

- I can make the second review directly in this PR by adding suggestions (this is the easiest way).
- I can create a PR that points to the branch in your fork (which is the source of the changes in this PR) which is the branch `IldySilva:maintenance_releases.Rmd-pt-auto`.

File: maintenance_releases.pt.Rmd [hunk 1]
```diff
@@ -3,7 +3,7 @@ aliases:
   - releasing.html
 ---
 
-# Liberação de um pacote {#releasing}
+# Publicação de um pacote {#releasing}
 
 ```{block, type="summaryblock"}
 Seu pacote deve ter versões diferentes ao longo do tempo: instantâneos de um estado do pacote que você pode lançar no CRAN, por exemplo. Essas versões devem ser devidamente _numeradas_, _lançadas_ e _descritas em um arquivo NEWS_. Mais detalhes abaixo.
```
Comment (pedropark99) [inline maintenance_releases.pt.Rmd:9]:
```suggestion
Seu pacote deve ter versões diferentes ao longo do tempo: *snapshots* de um estado do pacote que você pode publicar no CRAN, por exemplo. Essas versões devem ser devidamente _numeradas_, _publicadas_ e _descritas em um arquivo NEWS_. Mais detalhes abaixo.
```

File: maintenance_releases.pt.Rmd [hunk 2]
```diff
@@ -15,7 +15,7 @@ Observe que você pode simplificar o processo de atualização do NEWS e do cont
 
 - Recomendamos enfaticamente que os pacotes rOpenSci usem controle de versão semântico. Uma explicação detalhada está disponível na seção [capítulo de descrição](https://r-pkgs.org/description.html#version).
```
Comment (pedropark99) [inline maintenance_releases.pt.Rmd:16]: Aparently, the translation of the hyperlink did not worked properly in this line. This hyperlink should point to lifecycle chapter.

```suggestion
- Recomendamos enfaticamente que os pacotes rOpenSci usem controle de versão semântico. Uma explicação detalhada está disponível no [capítulo de ciclo de vida do livro *R Packages*](https://r-pkgs.org/lifecycle.html).
```

**Review by pedropark99 — CHANGES_REQUESTED** [review]
@IldySilva these are my suggestions as the second revisor.

File: maintenance_releases.pt.Rmd (hunk 2 above)
Comment (IldySilva) [reply maintenance_releases.pt.Rmd]: @pedropark99  but I see that in the documentation in English the link is the same (description)

Comment (pedropark99) [reply maintenance_releases.pt.Rmd]: The rOpenSci saw this problem with the link at the original content, and they fixed too. Now, we need to fix here too.

**Comment by pedropark99** [comment]
@yabellini   I understand that this PR is ready to be merged since both reviews are done.

**Pull request merged**